src/streaming.jl

```diff
@@ -215,7 +215,7 @@ function parsefile(filename::AbstractString,callbacks::XPCallbacks; bufferlines=
                 write(io, readline(file))
                 i += 1
             end
-            txt = Compat.bytestring(io)
+            txt = Compat.String(io)
```

Comment:
`@compat Compat.String` doesn't work on 0.3. As for dropping 0.3 support, let's ask @amitmurthy